chemicaltoolbox/rdock/rdock_macros.xml

```diff
@@ -1,5 +1,5 @@
 <macros>
-    <token name="@TOOL_VERSION@">2013.1-0</token>
+    <token name="@TOOL_VERSION@">2013.1</token>
```

Comment:
I'm not sure, if this number is higher than the previous one.

Reply:
Was the old version deleted? `conda search rdock` is only providing `2013.1`.

Anyway, getting the most recent version number is done with galaxy's conda_util, I didn't reimplement this.

Reply:
No clue how this could happen or ever worked. But this tool is either way deprecated left? Should we move it to deprecated?